docs/src/pages/[platform]/components/menu/demo.tsx

```diff
@@ -45,7 +48,7 @@ export const MenuDemo = () => {
       code={propsToCode(menuProps)}
       propControls={<MenuPropControls {...menuProps} />}
     >
-      <Menu {...menuProps}>
```

Comment:
Why this change? Shouldn't the destructuring work just as well?

Reply:
Destructuring doesn't work here because `useMenuProps` returns SetState hooks as well (e.g., `setSize`), which throws warnings in the console